### PR TITLE
Fix 411 release notes for incorrect issue verbiage, removed repeated phrases

### PR DESCRIPTION
**Pull request by tlblessing**
## Description

Fix 411 release notes for repeated phrases and incorrect verbiage (columns, not rows) in Oracle, PostgreSQL, and Redshift connectors.

### CODE REVIEW

**Comment by mosabua** [comment]
Was that a cut and paste and repeat error or is there some other wording needed @colebow ?

File: docs/src/main/sphinx/release/release-411.md [hunk 1]
```diff
@@ -122,17 +122,17 @@
 ## Oracle connector
 
 * Improve performance of queries when the network latency between Trino and
-  Oracle is high, or when selecting or when selecting a small number of rows. ({issue}`16644`)
+  Oracle is high, or when selecting a small number of rows. ({issue}`16644`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-411.md:125]:
```suggestion
  Oracle is high, or when selecting a small number of columns. ({issue}`16644`)
```

Comment (mosabua) [reply docs/src/main/sphinx/release/release-411.md:125]: Same for the others. I checked the original issue and this is the correct wording. cc @colebow

**Review by mosabua — CHANGES_REQUESTED** [review]
Please check for me in the original issue and confirm my update @colebow

File: docs/src/main/sphinx/release/release-411.md [hunk 2]
```diff
@@ -122,17 +122,19 @@
...
 * Improve performance of queries when the network latency between Trino and
-  Oracle is high, or when selecting or when selecting a small number of rows. ({issue}`16644`)
+  Oracle is high, or when selecting a small number of columns. ({issue}`16644`)
 
 ## PostgreSQL connector
 
 * Improve performance of queries when the network latency between Trino and
-  PostgreSQL is high, or when selecting or when selecting a small number of rows. ({issue}`16644`)
+  PostgreSQL is high, or when selecting a small number of columns.
+  ({issue}`16644`)
```
Comment (mosabua) [inline docs/src/main/sphinx/release/release-411.md:131]: Leave this on the prior line please (Trino release note style..)

**Review by colebow — APPROVED** [review]

**Review by mosabua — COMMENTED** [review]
Looks good. Thank you.

**Comment by mosabua** [comment]
Somehow PR still shows as changes requested .. but I approve.

**Pull request merged**